docker/tls/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# TLS job image 
+# Copyright (c) 2016-2018 ForgeRock AS.
+#
+# TLS job either creates an Self Signed certificate or adds imports user provided certificates
```

Comment:
s/Self Signed/self-signed/ (pls make this change throughout your comments)
Next line - s/Charts/charts/

Reply:
ah yes, will do